core/idl/wallet/ethereum/erc20.djinni

```diff
@@ -58,4 +58,7 @@ ERC20LikeOperation = interface +c {
     getOperationType(): OperationType;
     # Get opration status : pending or confirmed.
     getStatus(): i32;
+    # Get block height on which operation was included.
+    # @return Optional 64-bit integer, height of block in which operation was validated
+    getBlockHeight(): optional<i64>;
```

Comment:
Hm. Is it really needed to use an optional here? Do we care about a blockchain with no block vs. a blockchaini with only a single genesis block?

Also, djinni doesn’t have unsigned integers?!

Reply:
If not, I know what I have to do.

![](https://phaazon.net/media/uploads/eyebrows.gif)

Reply:
This is to allow clients (Live or Vault) to differentiate between them because they have a pending status, otherwise it will be problematic for them 
And yes no unsigned integers on djinni ... SFOL